### ISSUE TITLE
Validation Sets at Chip Level

### ISSUE THREAD

**Issue by jamesmcclain** [azavea/raster-vision#791]
Currently, there is the ability to produce validation sets at the scene level (that is if a chip is in the validation set, all chips generated from its originating scene are in the validation set [and the same for the training set]).

While this is convenient and legitimate, there are also legitimate cases in which one would like the sets to be shuffled at a finer level to reduce potential bias.

**Comment by AdeelH** [comment]
This is now possible via #1030
